### PR TITLE
Fixed Reset related ~line 43 Warning typos

### PR DESCRIPTION
**Pull request by linque1**

### CODE REVIEW

**Comment by linque1** [comment]
@medgarmedgar might not be using this account anymore due to a recent development.

**Comment by illfated** [comment]
Would you accept a whitespace improvement suggestion from me, although it might seem cosmetic to you?
I am constantly seeking to implement MarkDown indent marker compatibility spacing for the Note blobs (note, tip, important, warning, and so on).
Several of my own Pull Requests to Microsoft Docs are focused entirely on this specific subject.
Feel free to tell me if you require a MarkDown sourced reference for this spacing codestyle.

**Comment by linque1** [comment]
> Would you accept a whitespace improvement suggestion from me, although it might seem cosmetic to you?
> I am constantly seeking to implement MarkDown indent marker compatibility spacing for the Note blobs (note, tip, important, warning, and so on).
> Several of my own Pull Requests to Microsoft Docs are focused entirely on this specific subject.
> Feel free to tell me if you require a MarkDown sourced reference for this spacing codestyle.

I'm completely fine with that, I assume this will not affect the final output and I'll get a chance to preview before it's posted? If I'm The typos I fixed have been on for a while and I'd like them updated as soon as we can.

**Comment by illfated** [comment]
Sure, it is unrelated to your own changes nor will it affect those. The suggestion will be posed for you to approve & commit at your own leisure.

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Microsoft provides a [Windows Restricted Traffic Limited Functionality Baseline]
 >Regarding the Windows Restricted Traffic Limited Functionality Baseline, the 1903 settings (folder) are applicable to 1909 Windows >Enterprise devices. There were no additional settings required for the 1909 release.
 
 >[!Warning] 
->If a user executes the "Reset this PC" command (Settings -> Update & Security -> Recovery) with the "Keep my files" option (or the >"Remove Everything" option) the Windows Restricted Traffic Limited Functionality Baseline settings will need to be re-applied in order >re-restrict the device. Egress traffic may occur prior to the re-application of the Restricted Traffic Limited Functionality Baseline >settings.
+>If a user executes the **Reset this PC** command (Settings -> Update & Security -> Recovery) with the **Keep my files option** (or the **Remove Everything** option) the Windows Restricted Traffic Limited Functionality Baseline settings will need to be re-applied in order to re-restrict the device. Egress traffic may occur prior to the re-application of the Restricted Traffic Limited Functionality Baseline settings.
```
Comment (illfated) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:43]:
```suggestion
> If a user executes the **Reset this PC** command (Settings -> Update & Security -> Recovery) with the **Keep my files option** (or the **Remove Everything** option) the Windows Restricted Traffic Limited Functionality Baseline settings will need to be re-applied in order to re-restrict the device. Egress traffic may occur prior to the re-application of the Restricted Traffic Limited Functionality Baseline settings.
```
(MarkDown indent marker compatibility spacing)

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Microsoft provides a [Windows Restricted Traffic Limited Functionality Baseline]
 >Regarding the Windows Restricted Traffic Limited Functionality Baseline, the 1903 settings (folder) are applicable to 1909 Windows >Enterprise devices. There were no additional settings required for the 1909 release.
 
 >[!Warning]
```
Comment (illfated) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:42]:
```suggestion
> [!Warning] 
```
(MarkDown indent marker compatibility spacing)

**Comment by illfated** [comment]
Although there are quite a few inconsistencies regarding this specific MarkDown codestyle in this rather long page, I will refrain from suggesting any more changes. My goal here is mostly to spread the awareness about the single space between the MarkDown indent marker ( `>` ) and any following text being the de facto standard, whereas cases of a missing space is mentioned as "lazy coding" in MarkDown documentation.

**Comment by illfated** [comment]
Uhm, is this PR being closed to make place for a new PR in the private branches?

**Comment by linque1** [comment]
> Uhm, is this PR being closed to make place for a new PR in the private branches?

My apologies. My first time on the process. I just want this merged and change visible in public doc as soon as possible, my assumption was I needed to close this to have that happen?

**Comment by illfated** [comment]
Thank you. As you can see, the only way to get a PR published is to wait for one of the MS Docs maintainers or page managers to merge an open PR when it has been properly reviewed and deemed to be ready.

The next time you open a Pull Request and you are confident that all changes have been applied, simply comment with a single line saying only the following:

#sign-off

**Comment by illfated** [comment]
Also: Thank you for your contribution and the kind cooperation on my offer to improve your posted content.

**Comment by linque1** [comment]
> Thank you. As you can see, the only way to get a PR published is to wait for one of the MS Docs maintainers or page managers to merge an open PR when it has been properly reviewed and deemed to be ready.
> 
> The next time you open a Pull Request and you are confident that all changes have been applied, simply comment with a single line saying only the following:
> 
> #sign-off

Thanks. They have also communicated they will grant write permissions so I can merge my PRs.

**Comment by illfated** [comment]
Very well, that sounds great. I wish you all the best in your future work in the MSDocs private repository.

**Pull request merged**